azure/resource_groups/app/parameters/development.template.json

```diff
@@ -156,6 +156,22 @@
         "secretName": "GeckoboardAPIKey"
       }
     },
+    "LOGSTASH_HOST": {
+      "reference": {
+        "keyVault": {
+          "id": "${keyVaultId}"
+        },
+        "secretName": "LogstashHost"
+      }
+    },
+    "LOGSTASH_PORT": {
+      "reference": {
+        "keyVault": {
+          "id": "${keyVaultId}"
+        },
+        "secretName": "LogstashPort"
+      }
```

Comment:
I understand why the host needs to be a secret, but can the port just be defined in the parameters? Things in the KeyVault have a higher maintenance burden for the team so we should try to keep the amount of them down to a minimum.

Reply:
My logic was that it keeps the port and the host in the same place, but I'm easy on this if you think it makes more sense otherwise.

Reply:
Yeah that makes sense. Although I think having the value of the environment variables defined in the code where we can get away with it is probably better than keeping them together. If I was a future developer picking up a card to change the environment variables I would start with these ARM templates.